docs/source/conf.py

```diff
@@ -93,13 +95,22 @@ def is_stable(extra):
     tags.add('ipystable')
 else:
     tags.add('ipydev')
-    rst_prolog = """
-    .. warning::
+    rst_prolog += """
+.. warning::
+
+    This documentation is for a development version of IPython. There may be
```

Comment:
This documentation covers a development version of IPython. The development version may differ significantly from the latest stable release.